src/test/kotlin/lotto/MoneyTest.kt

```diff
@@ -0,0 +1,28 @@
+package lotto
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatIllegalArgumentException
+import org.junit.jupiter.api.Test
+import java.math.BigDecimal
+
+internal class MoneyTest {
+
+    @Test
+    fun `Money 객체 생성`() {
+        val money = Money(BigDecimal(1000))
+        assertThat(money.get).isEqualTo(1000)
+    }
+
+    @Test
+    fun `Money 객체 유효성 검사`() {
```

Comment:
```suggestion
    fun `음수 값이 들어오면 IllegalArgumentException이 발생한다`() {
```
어떤 경우에 유효성이 검증되는지 좀 더 구체적으로 작성해보면 어떨까요?